editor/property_selector.h

```diff
@@ -78,6 +78,7 @@ class PropertySelector : public ConfirmationDialog {
 	void select_property_from_instance(Object *p_instance, const String &p_current = "");
 
 	void set_type_filter(const Vector<Variant::Type> &p_type_filter);
+	void set_type_filter_exposable(const Vector<int> &p_type_filter);
```

Comment:
I think when we need to provide another method just for the bindings we usually use the `_bind` suffix, but it's not too important since this name won't be part of the public API. I think we also make these methods private and therefore prefixed with an underscore (`_`).

Reply:
as in `public set_type_filter` -> `private _set_type_filter` and `public set_type_filter_exposable` -> `public set_type_filter_bind` ?

Reply:
I'd say `private _set_type_filter_bind`

Bind methods shouldn't be public (they are private almost always, with some exceptions that I think are mistakes)

Reply:
Ah, I see. So `public set_type_filter` and `private _set_type_filter_bind` then.

Reply:
Yep, it's not uniform but private methods should be prefixed by `_`, they should be private to keep them internal, and communicating that it is a bind method is just a good practice to avoid any confusion.